### PR TITLE
feat : Cards template #30

### PR DESCRIPTION
**Pull request by juldpnt**
### **What has been done ?**
<hr>

This PR aims to tackle the issue [ Update "Qui sommes nous ?" section #30 ](https://github.com/pyronear/pyronear.github.io/issues/30) taking inspiration from the figma mockup therein.

Bootstrap cards template could not be used as it's only available for BS >= 4 and we are using BS 3. So I ended up with custom flexbox objects as the migration towards v4 had some unexpected behaviours.

**About us section**:
- Split phrases into title and content
- Add icons
- Add card structure

**Css in general**:
- Modify theme-primary and add theme-secondary to fit the figma file
- Add responsive card design 
- Add :hover behaviour to buttons, this was not 
- Add highlighter design to Services section ("Comment ça marche ?") by replacing bold behaviour

**Rest**:
- Fix color changes after theme color modification

### **What needs to be checked ?**
<hr>
I tried at best to take the same icons as the ones in the figma file.I'm not used to license issues, so I would appreciate if someone more into it could confirm that everything is in order. 

All icon sources and licenses are specified in **img/cards_icons/sources.txt**. They are under **MIT**, **CC BY 4.0** and **Apache 2** licenses, but I do have some doubts about Apache 2 usage... should we change these icons ?

Do not hesitate to remove the sources.txt file if not necessary anymore.

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -223,13 +244,34 @@ t:
...
-      phrase5: "<b>Automatic</b>: We offer a fully automated solution to free up firefighters' time."
-      phrase6: "<b>Energetically sober</b>: We are in a so-called low-tech approach, we have built a low-consumption device allowing an easier deployment and a more reasoned use of our precious resources."
-      phrase7: "<b>Economic</b>: Our R&D is done by volunteers, our device is sober and the non-profit object of our association, which allows us to propose very competitive prices to facilitate a deployment on the territory."
-      phrase8: "<b>Modular</b>: We offer a complete detection system but our device has been designed to work with existing equipment (use of cameras already in place, connection with existing software ...)"
+      card1:
+        icon_src: "/img/cards_icons/opensource.svg"
+        title: "Open source"
+        content: "<b>Open source</b>: All our code is available for free and co-developed by our volunteers. We believe that collective intelligence is the best way to solve a complex problem."
```
Comment (fe51) [inline _config.yml:250]: Typo, remove ```<b>Open source</b>``` in content as it is already in the "title"

File: _config.yml [hunk 2]
```diff
@@ -223,13 +244,34 @@ t:
...
+        title: "Energetically sober"
+        content: "We are in a so-called low-tech approach, we have built a low-consumption device allowing an easier deployment and a more reasoned use of our precious resources."
+      card6:
+        icon_src: "/img/cards_icons/savings-solid.svg"
+        title: "Economic" 
+        content: "Our R&D is done by volunteers, our device is sober and the non-profit object of our association, which allows us to propose very competitive prices to facilitate a deployment on the territory."
+      card7:
+        icon_src: "/img/cards_icons/grid-view-solid.svg"
+        title: "Modular"
+        content: "We offer a complete detection system but our device has been designed to work with existing equipment (use of cameras already in place, connection with existing software ...)"
```
Comment (fe51) [inline _config.yml:274]: Maybe more clear
```suggestion
        content: "We offer a complete detection pipeline but our system has been designed to work with existing equipment (use of cameras already in place, connection with existing software ...)"
```

File: img/cards_icons/sources.txt
Comment (fe51) [inline img/cards_icons/sources.txt:1]: For sure, icon licences you have chosen can be used here. 

As you stated in the pitch of the PR, I am not sure whether source.txt is required, but I think is not an issue to have it here (a deletion or refactor could be done later)

**Review by fe51 — CHANGES_REQUESTED** [review]
Hi there, Thanks a lot for the PR that will significantly improve the website !

Just a few comments, otherwise seems OK to merge ! 

Which tools or website have you been using for translation  ? Unless you are fluent in Catalan :D

File: _config.yml (hunk 1 above)
Comment (juldpnt) [reply _config.yml:250]: I did not translate, theses parts were already done in previous PR ! I did only add the icon_src and split the phrase into title and content.

**Comment by juldpnt** [comment]
Thank you for the comments, I modified the typo you mentionned earlier in this commit !

**Review by fe51 — APPROVED** [review]

**Comment by fe51** [comment]
Cool, thanks for the updates ! Let's merge :)

**Comment by fe51** [comment]
closes #30

**Pull request merged**